frontend/Interfaces/buddy/DIP/imgcodecs/Replenishment.hpp

```diff
@@ -0,0 +1,148 @@
+

```

Comment:
Add opencv's license here.
Add a comment stating that the file was modified by us based on the xxx file of opencv.